src/index.js

```diff
@@ -16,22 +16,26 @@ function makeUrlAbsolute(url) {
 class WebSdkApi extends EventEmitter {
   constructor(child) {
     super();
-    this.available = () => {}; // Starts as a no-op, overwritten as a resolve once .ready is called
     this.child = child;
     child.msg_bus.on("alert", (event, ...args) => this.emit(event));
     child.msg_bus.on("signal", signal => this.emit('signal', signal));
     child.msg_bus.on("unrecoverable-agent-state", () => this.emit('unrecoverable-agent-state'));
-    child.msg_bus.on("unavailable", () => this.emit('unavailable'));
-    child.msg_bus.on("available", () => {
-      this.available()
+    child.msg_bus.on("unavailable", () => {
+      this.isAvailable = false
```

Comment:
```suggestion
      this.is_available = false
```
If I remember correctly, our agreement was to make internal vars use snake case.  If that's no longer the case, can we update the ReadMe to show the correct casing.